x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/application/EnterpriseSearch.java

```diff
@@ -272,7 +276,8 @@ public List<RestHandler> getRestHandlers(
                     new RestPostConnectorSyncJobAction(),
                     new RestDeleteConnectorSyncJobAction(),
                     new RestCancelConnectorSyncJobAction(),
-                    new RestCheckInConnectorSyncJobAction()
+                    new RestCheckInConnectorSyncJobAction(),
```

Comment:
Nit for dealing with merge conflicts, let's try to keep it ordered alphabetically Cancel < Check < Delete. WYT?

Reply:
Fair point, actually I thought I've ordered it in a different PR, but probably forgot it here.